specification/databricks/resource-manager/Microsoft.Databricks/stable/2018-04-01/databricks.json

```diff
@@ -0,0 +1,467 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "DatabricksClient",
+    "version": "2018-04-01",
+    "description": "ARM Databricks"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
```

Comment:
Few APIs missing -
1. Listing all workspaces under a subscription
2. Listing all workspaces under a resource group
3. operations API to list all the available operations/actions for Microsoft.Databricks. Described here- https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/proxy-api-reference.md#exposing-available-operations

Reply:
I just added 1 and 2. For 3 (i.e. operations API), we are still working on the backend implementation and planning to add the swagger doc for that later.